common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -283,6 +305,14 @@ def test_deferred_verification_enrollment(self):
         # Submit payment
         self.fake_payment_page.submit_payment()
 
+        # Expect enrollment activated event
+        assert_event_emitted_num_times(
```

Comment:
I've enabled that test locally and the answer is no.... That event doesn't appear until after submit_payment.